elements/pfe-button/docs/pfe-button.md

```diff
@@ -15,46 +15,40 @@ import '@patternfly/pfe-icon';
 
 #### Default - Medium size
 
-<pfe-button><button>Primary</button></pfe-button>
-<pfe-button variant="secondary"><button>Secondary</button></pfe-button>
-<pfe-button variant="secondary" danger><button>Danger Secondary</button></pfe-button>
-<pfe-button variant="tertiary"><button>Tertiary</button></pfe-button>
-<pfe-button danger><button>Danger</button></pfe-button>
-<pfe-button warning><button>Warning</button></pfe-button>
+<pfe-button>Primary</pfe-button>
+<pfe-button variant="secondary">Secondary</pfe-button>
+<pfe-button variant="secondary" danger>Danger Secondary</pfe-button>
+<pfe-button variant="tertiary">Tertiary</pfe-button>
+<pfe-button danger>Danger</pfe-button>
+<pfe-button warning>Warning</pfe-button>
 
 #### Link variant
 
 <pfe-button variant="link" id="test">
   <svg slot="icon" fill="currentColor" style="vertical-align:-0.125em" height="1em" width="1em" viewBox="0 0 512 512" aria-hidden="true">
     <path d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm144 276c0 6.6-5.4 12-12 12h-92v92c0 6.6-5.4 12-12 12h-56c-6.6 0-12-5.4-12-12v-92h-92c-6.6 0-12-5.4-12-12v-56c0-6.6 5.4-12 12-12h92v-92c0-6.6 5.4-12 12-12h56c6.6 0 12 5.4 12 12v92h92c6.6 0 12 5.4 12 12v56z"></path>
   </svg>
-  <button>Link</button>
+  Link
 </pfe-button>
-<pfe-button variant="link" icon="fa-external-link-square" icon-position="right">
-  <button>Link</button>
-</pfe-button>
-<pfe-button variant="link" inline><button>Inline Link</button></pfe-button>
-<pfe-button variant="link" danger><button>Danger Link</button></pfe-button>
+<pfe-button variant="link" icon="fa-external-link-square" icon-position="right">Link</pfe-button>
+<pfe-button variant="link" inline>Inline Link</pfe-button>
+<pfe-button variant="link" danger>Danger Link</pfe-button>
 
 #### Plain button
 
 <pfe-button plain>
-  <button aria-label="Action">
-    <svg fill="currentColor" height="1em" width="1em" viewBox="0 0 352 512">
-      <path d="M242.72 256l100.07-100.07c12.28-12.28 12.28-32.19 0-44.48l-22.24-22.24c-12.28-12.28-32.19-12.28-44.48 0L176 189.28 75.93 89.21c-12.28-12.28-32.19-12.28-44.48 0L9.21 111.45c-12.28 12.28-12.28 32.19 0 44.48L109.28 256 9.21 356.07c-12.28 12.28-12.28 32.19 0 44.48l22.24 22.24c12.28 12.28 32.2 12.28 44.48 0L176 322.72l100.07 100.07c12.28 12.28 32.2 12.28 44.48 0l22.24-22.24c12.28-12.28 12.28-32.19 0-44.48L242.72 256z"></path>
-    </svg>
-  </button>
+  <svg aria-label="Action" fill="currentColor" height="1em" width="1em" viewBox="0 0 352 512">
```

Comment:
Use offscreen text and aria-hidden on SVG. See https://www.sarasoueidan.com/blog/accessible-icon-buttons/